documentation-site/components/sidebar.js

```diff
@@ -50,13 +43,31 @@ const NavigationLink = props => {
 };
 
 const NavigationItem = props => {
-  const {route, index, level = 1} = props;
+  const {route, index, level = 1, path} = props;
   return (
     <Block
       font={levelToFont[level]}
-      paddingLeft={levelToPadding[level]}
       paddingBottom={level !== 3 ? 'scale100' : 0}
-      paddingTop={index === 0 && level === 3 ? 'scale100' : 0}
+      marginTop={index === 0 && level === 3 ? 'scale300' : 'scale300'}
+      overrides={{
+        Block: {
+          style: ({$theme}) => ({
+            paddingLeft: !route.children ? $theme.sizing.scale600 : '0px',
+            borderLeft:
+              (path && path === route.path) || (!path && route.path === '/')
```

Comment:
Do we need the first `path` in `path && path === route.path`?

Reply:
```suggestion
              (path === route.path) || (!path && route.path === '/')
```

Reply:
This is actually needed because `route.path` can be undefined